example/app/models.py

```diff
@@ -16,7 +16,9 @@ class ModelWithFiles(models.Model):
         null=True,
         s3_config=saritasa_s3_tools.S3FileTypeConfig(  # pyright: ignore
             name="django-files",
-            key=saritasa_s3_tools.keys.S3KeyWithPrefix("django-files"),
+            key=saritasa_s3_tools.keys.S3KeyWithPrefixUUIDFolder(
```

Comment:
Maybe remove `S3Key` prefix? class already located in saritasa**s3**tools, **keys** module